README.md

```diff
@@ -68,6 +68,9 @@ Attributes
 * `ntp['peer']['maxpoll']` (applies to NTP Servers ONLY)
   - Boolean. Defaults to 10 (ntp default). Specify the maximum poll intervals for NTP messages, in seconds to the power of two.
 
+* `ntp['server']['prefer']` (applies to NTP Servers and Clients)
+  - String. No Default. The server from `ntp['servers']` to prefer getting the time from.
```

Comment:
I was curious as to what the standard in this case as I didn't see any empty string defaults already existing in the README.  I am used to specifying the default value explicitly but
I guess I took the functional description this time.  
"No Default *preferred server*" instead of "No Default *value*"